stories/topics/proc-azure-monitor-deployment-engine.adoc

```diff
@@ -18,6 +18,11 @@ Approximately 10 minutes into the deployment process, the *Outputs* section of t
 . Login to the deployment using the following credentials:
   * *Username*: _admin_
   * *Password*: Use the _Administrator Password_ that you chose when configuring your deployment.
+. Login to your Red Hat account.
```

Comment:
Can we add more information to this step? At this point, the user is on the deployment driver web UI page and is looking at the following modal dialog so it would be great to mention it somehow.
![Screenshot from 2023-12-04 07-12-50](https://github.com/ansible/aap-docs/assets/93541722/80b616ee-64be-45f8-ba89-652ff6868e2b)

Reply:
@tznamena , I'll add more context.